deployment/k8s/helmfile/values-dev.yaml

```diff
@@ -0,0 +1,172 @@
+conditionalPackageInstalls:
+  snapshotVolume:
+    enabled: true
```

Comment:
Do these need this many layers?  Could we do w/ 2?

Reply:
This was a weird requirement that I had from the helmfile to structure this way at least to my knowledge